packages/opencensus-propagation-stackdriver/test/test-stackdriver-format.ts

```diff
@@ -0,0 +1,145 @@
+/**
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {HeaderGetter, HeaderSetter} from '@opencensus/core';
+import * as assert from 'assert';
+import {StackdriverFormat, TRACE_CONTEXT_HEADER_NAME} from '../src';
+
+const stackdriverFormat = new StackdriverFormat();
+
+describe('StackdriverPropagation', () => {
+  describe('extract()', () => {
+    it('should extract context of a sampled span from headers', () => {
+      const headers: {[key: string]: string|string[]|undefined} = {};
+      headers[TRACE_CONTEXT_HEADER_NAME] = '123456/667;o=1';
+      const getter: HeaderGetter = {
```

Comment:
Would it make sense to abstract this into a helper function to share it between these different test cases?

Reply:
Sure, done